<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScheduleDialog</class>
 <widget class="QDialog" name="ScheduleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>391</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Manage Schedule</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>231</width>
     <height>348</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="UnitLayout">
    <item>
     <widget class="QTabWidget" name="scheduleListContainer">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>170</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="scheduleList">
       <attribute name="title">
        <string>Schedule List</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>221</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="scheduleHList">
         <item>
          <widget class="QListView" name="scheduleListView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="scheduleVList">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="scheduleAddList">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="scheduleRemoveList">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="ItemsContainer">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>170</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Presets">
       <attribute name="title">
        <string>Presets</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>221</width>
          <height>141</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="presetsHContainer">
         <item>
          <widget class="QListView" name="presetsListView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="presetsVContainer">
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="presetUpperSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="presetsAddButton">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="presetBottomSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>501</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="scheduleSetLayout">
    <item>
     <widget class="QTabWidget" name="scheduleContentContainer">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>170</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="scheduleContent">
       <attribute name="title">
        <string>Schedule Content</string>
       </attribute>
       <widget class="QTableView" name="scheduleTableView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>501</width>
          <height>151</height>
         </rect>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::SelectedClicked</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="schedulerContainer">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>170</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="schedulerView">
       <attribute name="title">
        <string>Scheduler</string>
       </attribute>
       <widget class="QTimeEdit" name="scheduleBeganTime">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>20</y>
          <width>118</width>
          <height>22</height>
         </rect>
        </property>
        <property name="displayFormat">
         <string>h:mm:ss AP</string>
        </property>
       </widget>
       <widget class="QTimeEdit" name="scheduleEndTime">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>20</y>
          <width>118</width>
          <height>22</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>h:mm:ss AP</string>
        </property>
       </widget>
       <widget class="QLabel" name="scheduleBeganLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>47</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Schedule 
Began</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="scheduleEndLabel">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>10</y>
          <width>51</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Estimated 
End</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="scheduleDurationLabel">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Duration </string>
        </property>
       </widget>
       <widget class="QLabel" name="scheduleDuration">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>60</y>
          <width>301</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>TBD</string>
        </property>
       </widget>
       <widget class="QLabel" name="scheduleActivatedLabel">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Activated </string>
        </property>
       </widget>
       <widget class="QLabel" name="scheduleActivated">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>80</y>
          <width>301</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Not Activated</string>
        </property>
       </widget>
       <widget class="QPushButton" name="scheduleActivateButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>110</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Activate Schedule</string>
        </property>
       </widget>
       <widget class="QPushButton" name="scheduleDeactivateButton">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>110</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Deactivate Schedule</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>370</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
  <widget class="QLabel" name="selectedScheduleNameLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>411</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Editing Schedule ''</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
